sample/HelloWorld/public/js/helloworld.js

```diff
@@ -6,8 +6,6 @@ var publisher = OT.initPublisher('publisher', {
     insertMode: 'append',
     width: '100%',
     height: '100%'
-}, function(error) {
-  alert(error);
 });
```

Comment:
Wouldn't that be interesting to keep this callback? I feel like a lot of people do have camera/microphone permission issues.